dags/multipod/configs/maxtext_sweep_gce_config.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2023 Google LLC
```

Comment:
Do you think we could consolidate the GCE and GKE files? It looks like we could make the parameter sweep logic shared between the two and just change how we construct the `Task` as either `TpuGkeTask` or `TpuGceTask`.

Reply:
I think we could consolidate the files and parameter sweep logic, but I think we would still need 2 separate functions since there are arguments unique to just GCE (runtime_version, etc.) and GKE (cluster_name, etc.)

Reply:
Yeah, that makes sense to me. It would be nice to centralize the parameter sweeping logic since there's so much overlap, but if it makes the implementation too clunky it's might not be worth it.

Reply:
I will also add this in a later PR. I am planning on adding some more sweep logic and will consolidate it then.